cmd/client-url.go

```diff
@@ -228,7 +228,12 @@ func isURLPrefixExists(urlPrefix string, incomplete bool) bool {
 // on failure just return 'application/octet-stream'.
 func guessURLContentType(urlStr string) string {
 	url := newClientURL(urlStr)
-	contentType := mime.TypeByExtension(filepath.Ext(url.Path))
+	var contentType string
+	if objectExt := filepath.Ext(url.Path); objectExt != "" {
```

Comment:
can you create a function similar to mimedb.TypeByExtension? it would be better original package has this function. This can help avoid the large modifications on the `mc` end, making sure that `mimedb` is a drop in replacement for `mime` package.

Reply:
ok . Will send a PR to minio/minio and then re-vendorize this